### PR TITLE
kishore commits

### PR DESCRIPTION
**Pull request by ykishorek**

### CODE REVIEW

File: abc.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# /bin/bash
+echo "this file ios created by kishore"
+eco "this file has merig conflicts"
```
Comment (santhu3064) [inline abc.sh]: Typo its echo

File: revanth.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
 echo "this file is created by revanth"
+echo "This file is update by kishore"
```
Comment (santhu3064) [inline revanth.sh]: updated*

**Review by santhu3064 — CHANGES_REQUESTED** [review]

File: abc.sh (hunk 1 above)
Comment (ykishorek) [reply abc.sh]: updateed

File: revanth.sh (hunk 2 above)
Comment (ykishorek) [reply revanth.sh]: updated

**Review by santhu3064 — APPROVED** [review]

**Pull request merged**